bower_components/materialize/sass/components/_material-icons.scss

```diff
@@ -0,0 +1,194 @@
+$font-material-icons-name   : 'Material Icons';
+$font-material-icons-file   : 'MaterialIcons-Regular';
+
+@font-face {
+    font-family: "#{$font-material-icons-name}";
+    font-weight: 400;
+    font-style: normal;
+    src: url("#{$icons-font-path}#{$font-material-icons-file}.eot"); /* For IE6-8 */
+    src: local("#{$font-material-icons-name}"),
+         local("#{$font-material-icons-file}"),
+         url("#{$icons-font-path}#{$font-material-icons-file}.woff2") format("woff2"),
+         url("#{$icons-font-path}#{$font-material-icons-file}.woff") format("woff"),
+         url("#{$icons-font-path}#{$font-material-icons-file}.ttf") format("truetype");
+}
+
+.material-icons {
+    font-family: 'Material Icons';
+    font-weight: normal;
+    font-style: normal;
+    font-size: 24px;  /* Preferred icon size */
+    display: inline-block;
+    width: 1em;
+    height: 1em;
+    line-height: 1;
+    text-transform: none;
+    letter-spacing: normal;
+    word-wrap: normal;
+
+    /* Support for all WebKit browsers. */
+    -webkit-font-smoothing: antialiased;
+    -webkit-font-feature-settings: 'liga';
+
+    /* Support for Safari and Chrome. */
+    text-rendering: optimizeLegibility;
+
+    /* Support for Firefox. */
+    -moz-osx-font-smoothing: grayscale;
+
+    /* Support for IE. */
+    font-feature-settings: 'liga';
+
+    // Rules for sizing the icon.
+    &.md-18 { font-size: 18px; }
+    &.md-24 { font-size: 24px; }
+    &.md-36 { font-size: 36px; }
+    &.md-48 { font-size: 48px; }
+
+    // Rules for using icons as black on a light background.
+    &.md-dark { color: rgba(0, 0, 0, 0.54); }
+    &.md-dark.md-inactive { color: rgba(0, 0, 0, 0.26); }
+
+    // Rules for using icons as white on a dark background.
+    &.md-light { color: rgba(255, 255, 255, 1); }
+    &.md-light.md-inactive { color: rgba(255, 255, 255, 0.3); }
+
+    /* rules taken from deprecated material-design-icons */
```

Comment:
This are NEW rules from google (see Paragrah: Styling icons in material design in https://google.github.io/material-design-icons/) 

The rules AFTER comment are taken from materialize-sass like SPIN, ROTATE etc. and they are mostly useful - imho